_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -98,52 +113,56 @@ On the VM/external host:
 dig istio-pilot.istio-system
 ```
 ```
-    # This should be the same address shown as "EXTERNAL-IP" in 'kubectl get svc -n istio-system istio-pilot-ilb'
-    ...
-    istio-pilot.istio-system. 0	IN	A	10.128.0.5
-    ...
+# This should be the same address shown as "EXTERNAL-IP" in 'kubectl get svc -n istio-system istio-pilot-ilb'
+...
+istio-pilot.istio-system. 0	IN	A	10.128.0.5
+...
 ```
 ```bash
-    # Check that we can resolve cluster IPs. The actual IN A will depend on cluster configuration.
-    dig istio-pilot.istio-system.svc.cluster.local.
+# Check that we can resolve cluster IPs. The actual IN A will depend on cluster configuration.
+dig istio-pilot.istio-system.svc.cluster.local.
 ```
 ```
-    ...
-    istio-pilot.istio-system.svc.cluster.local. 30 IN A 10.23.251.121
+...
+istio-pilot.istio-system.svc.cluster.local. 30 IN A 10.23.251.121
 ```
 ```bash
 dig istio-ingress.istio-system.svc.cluster.local.
 ```
 ```
-    ...
-    istio-ingress.istio-system.svc.cluster.local. 30 IN A 10.23.245.11
+...
+istio-ingress.istio-system.svc.cluster.local. 30 IN A 10.23.245.11
 ```
 
 * Verify connectivity by checking whether the VM can connect to Pilot and to an endpoint.
 
-    ```bash
-    curl -v 'http://istio-pilot.istio-system:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
-    ```
-    ```
-    ...
-    "ip_address": "10.20.1.18",
-    ...
-    ```
-    ```bash
-    # On the VM: use the address above - it will connect directly the the pod running istio-pilot.
-    curl -v 'http://10.20.1.18:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
-    ```
+```bash
+curl -v 'http://istio-pilot.istio-system:8080/v1/registration/istio-pilot.istio-system.svc.cluster.local|http-discovery'
+```
+```
+...
+"ip_address": "10.20.1.18",
+...
+```
+```bash
+# On the VM: use the address above - it will connect directly the the pod running istio-pilot.
```

Comment:
On the VM, use the address above. It will directly connect the pod ...

Reply:
thx, I'm adding "to" in 'connect to the pod' too